File: LV0/주사위 게임 2.js
function solution(a, b, c) {
  let sum = a + b + c;
  let sumOfSquares = a * a + b * b + c * c;
  let sumOfCubes = a * a * a + b * b * b + c * c * c;

  if (a !== b && b !== c && a !== c) {
    return sum;
  } else if (a === b && b === c) {
    return sum * sumOfSquares * sumOfCubes;
  } else {
    return sum * sumOfSquares;
  }
}

File: LV0/주사위 게임 2.java
class Solution {

  public long solution(int a, int b, int c) {
    long answer = 0;
    long sum = a + b + c;
    long sumOfSquares = a * a + b * b + c * c;
    long sumOfCubes = a * a * a + b * b * b + c * c * c;

    if (a != b && b != c && a != c) {
      answer = sum;
    } else if (a == b && b == c) {
      answer = sum * sumOfSquares * sumOfCubes;
    } else {
      answer = sum * sumOfSquares;
    }
    return answer;
  }
}
